.github/workflows/ci.yml

```
@@ -1227,8 +1227,16 @@ jobs:
                     sudo apt-get update
                     sudo apt-get install -y pvs-studio-dotnet
 
-            -   name: Restore
-                run: dotnet restore src/squirix/Squirix.csproj src/squirix.server/Squirix.Server.csproj src/squirix.server.host/Squirix.Server.Host.csproj src/squirix.protocol-model/Squirix.ProtocolModel.csproj src/squirix.analyzers/Squirix.Analyzers.csproj
+            -   name: Restore production projects
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)